src/app/modals/add-to-group/add-to-group.component.ts

```diff
@@ -244,13 +246,18 @@ export class AddToGroupModal implements OnInit, OnDestroy {
 
   public searchGroups(event?): void {
     // Execute search after every third character of the search term
```

Comment:
I think we should remove this comment - its kind of misleading with the current implementation!